### PR TITLE
[dartdev] remove dart from flutter/bin

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Remove bin/dart, bin/dart.bat files. With the removal of dartdev from the next release, there is little reason to want the `dart` binary on your path instead of dart-sdk/bin, since only the latter of which has the snapshots/pub executables needed to develop an actual dart program.

### CODE REVIEW

File: bin/internal/shared.bat [hunk 1]
```diff
@@ -157,4 +157,7 @@ GOTO :after_subroutine
   REM Exit Subroutine
   EXIT /B
 
+:final_exit
```
Comment (zanderso) [inline bin/internal/shared.bat:160]: What's the reason for moving this here?

**Comment by zanderso** [comment]
fyi @pcsosinski

File: bin/internal/shared.bat (hunk 1 above)
Comment (jonahwilliams) [reply bin/internal/shared.bat:160]: This is actually a bug, we can't jump to this label from shared.bat

**Comment by pcsosinski** [comment]
just to confirm we decided not to do this in 1.20 right?

**Comment by zanderso** [comment]
That's right.

**Pull request closed without merging**